erpnext/stock/doctype/packed_item/packed_item.json

```diff
@@ -523,6 +523,32 @@
    "search_index": 0, 
    "set_only_once": 0, 
    "unique": 0
+  },
+  {
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "fieldname": "packing_description", 
```

Comment:
Why do you need to add a new field when there is already a field called `description` in Packed Item doctype?